### ISSUE TITLE
Potential 1.3.1 Fail: Selected Life Journey Stage does not have a corresponding heading or section markup

### ISSUE THREAD

**Issue by neilmispelaar** [DTS-STN/Life-Journeys#100]
### Description

A visual user (i.e. not using a screen reader) can see that when they select the dropdown and choose a life journey stage (for example "Child's arrival"), there is a visual change in the content directly below the select box that occurs, and they can see that the loaded content is grouped together (because the select box and loaded content is close together you can make an assumption that the loaded content is "Child's arrival" content). 

For users with screen readers however, they are unable to perceive this grouping, nor do they have a heading at the start of the loaded content to provide information on what document section they are about to read, or what exactly was loaded in. 

The select option they may have interacted with should not be used in place of a document heading.

<img width="1632" alt="Screen Shot 2021-08-08 at 4 46 37 PM" src="https://user-images.githubusercontent.com/9325038/128645364-f9db1a42-3a52-4926-8357-0c5cd8ea31d2.png">

### How to test 

1. Load the project page 
2. Scroll down to the "Discover the stages of the new child life journey" area on the page
3. Activate the drop down and select any state other than the default state, for example "Child's Arrival"
4. The life journey content will load
5. Inspect the page
6. You can't find a heading that marks the start of the "Child's arrival" content
7. Repeat for each life journey and default state

### How to fix 

Add a heading and section tags around the content that is loaded in by the `select` box as well as a heading for the `default` content. 

```
<section>
   <h2>Discover the stages of the new child life journey</h2>
   
   <select>
      .... <!-- Select box set to "Child's arrival" --> 
   </select>
   
   <section>
      <h3 class="sr-only">Child's arrival</h3>
         <!-- Begin Child's arrival content --> 
         <section>
            <h4>Birth</h4>
            ...
         </section>
         ...
         <!-- End child's arrival content --> 
   </section>  <!-- End of h3 --> 
   
</section> <!-- End of h2 --> 
```

Note: with the exception of the `h1` that is wrapped by a `main` tag, every heading element should be wrapped with a grouping element such as a `section` tag. Other grouping tags are `nav` `aside` `header` `footer` but you don't normally use those within `main`
 
### Resources

[WGAC 1.3.1 - Understanding](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
[H42 - Using h1-h6 to identify headings](https://www.w3.org/WAI/WCAG21/Techniques/html/H42)
[G141 - Organizing a page using headings](https://www.w3.org/WAI/WCAG21/Techniques/general/G141)
[WebAIM - Semantic Structure](https://webaim.org/techniques/semanticstructure/)
